hpe3par_sdk/client.py

```diff
@@ -2604,7 +2620,37 @@ def getRemoteCopyGroup(self, name):
         :returns: Remote Copy Group
 
         """
-        return self.client.getRemoteCopyGroup(name)
+        return RemoteCopyGroup(self.client.getRemoteCopyGroup(name))
+
+    def getRemoteCopyGroupVolumes(self, remoteCopyGroupName):
+        """
+        Returns information on all volumes in a Remote Copy Groups
+
+        :param remoteCopyGroupName: the remote copy group name
+        :type name: str
```

Comment:
Correct the type <parameter> syntax in docstring here and rest other places of this PR.